rust/agents/validator/src/submit.rs

```diff
@@ -235,86 +235,6 @@ impl ValidatorSubmitter {
         }
         Ok(())
     }
-
-    pub(crate) async fn legacy_checkpoint_submitter(self) -> Result<()> {
-        // current_index will be None if the validator cannot find
-        // a previously signed checkpoint
-        let mut current_index = self.checkpoint_syncer.latest_index().await?;
-
-        if let Some(current_index) = current_index {
-            self.metrics
-                .legacy_latest_checkpoint_processed
-                .set(current_index as i64);
-        }
-
-        // How often to log checkpoint info - once every minute
-        let checkpoint_info_log_period = Duration::from_secs(60);
-        // The instant in which we last logged checkpoint info, if at all
-        let mut latest_checkpoint_info_log: Option<Instant> = None;
-        // Returns whether checkpoint info should be logged based off the
-        // checkpoint_info_log_period having elapsed since the last log.
-        // Sets latest_checkpoint_info_log to the current instant if true.
-        let mut should_log_checkpoint_info = || {
-            if let Some(instant) = latest_checkpoint_info_log {
-                if instant.elapsed() < checkpoint_info_log_period {
-                    return false;
-                }
-            }
-            latest_checkpoint_info_log = Some(Instant::now());
-            true
-        };
-
-        loop {
-            // Check the latest checkpoint
-            let latest_checkpoint = self
-                .merkle_tree_hook
-                .latest_checkpoint(self.reorg_period)
-                .await?;
-
-            self.metrics
-                .legacy_latest_checkpoint_observed
-                .set(latest_checkpoint.index as i64);
-
-            // Occasional info to make it clear to a validator operator whether things are
```

Comment:
Can we keep the occasional logs added in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/1112 ? They're not in the new submitter, but we got a lot of feedback from people that without it they weren't sure if the validator was actually working because of long periods between messages